pkg/runner/job_executor.go

```diff
@@ -47,9 +63,10 @@ func newJobExecutor(info jobInfo) common.Executor {
 			}
 			return nil
 		})
-	}
 
-	steps = append(steps, func(ctx context.Context) error {
+		postSteps = append(postSteps, step.post())
```

Comment:
You messed up the order of post steps? They are called in reverse order see my other comment.

Reply:
Done